.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish Documentation Site
+
+on:
+  push:
+    branches: [stromberg/manual-publish]
```

Comment:
I assume this is just here for testing this PR?

Reply:
Yep. Just for testing.

I actually created a separate repo on my personal account to avoid spamming everyone on this repo with build failures. I'll update the PR when all of that is working as intended.